content/docs/rendering-elements.md

```diff
@@ -18,58 +18,58 @@ const element = <h1>Hola, mundo</h1>;
 
 A diferencia de los elementos del DOM de los navegadores, los elementos de React son objetos planos, y su creación es de bajo costo. React DOM se encarga de actualizar el DOM para igualar los elementos de React.
 
->**Note:**
+>**Nota:**
 >
->One might confuse elements with a more widely known concept of "components". We will introduce components in the [next section](/docs/components-and-props.html). Elements are what components are "made of", and we encourage you to read this section before jumping ahead.
+>Uno podría confundir los elementos con el muy conocido concepto de "componentes". En la siguiente sección [siguiente sección](/docs/components-and-props.html) hablaremos de componentes. Los elementos son los que "constituyen" los componentes, y recomendamos leer esta sección antes de continuar.
 
-## Rendering an Element into the DOM
+## Renderizando un Elemento en el DOM
 
-Let's say there is a `<div>` somewhere in your HTML file:
+Digamos que hay un `<div>` en alguna parte de tu archivo HTML:
 
 ```html
 <div id="root"></div>
 ```
 
-We call this a "root" DOM node because everything inside it will be managed by React DOM.
+Nosotros lo llamamos un nodo "raíz" porque todo lo que esté dentro de él será manejado por React DOM.
 
-Applications built with just React usually have a single root DOM node. If you are integrating React into an existing app, you may have as many isolated root DOM nodes as you like.
+Las aplicaciones construidas con solo React usualmente tienen un único nodo raíz en el DOM. Dado el caso que estés integrando React en una aplicación existente, podrías tener cuantos nodos raíz en el DOM como quieras.
 
-To render a React element into a root DOM node, pass both to `ReactDOM.render()`:
+Para renderizar un elemento de React en el nodo raíz del DOM, ambos se pasan a `ReactDOM.render()`:
 
 `embed:rendering-elements/render-an-element.js`
 
 [](codepen://rendering-elements/render-an-element)
 
-It displays "Hello, world" on the page.
+Esto muestra "Hello, world" en la página.
 
-## Updating the Rendered Element
+## Actualizando el Elemento Renderizado
 
-React elements are [immutable](https://en.wikipedia.org/wiki/Immutable_object). Once you create an element, you can't change its children or attributes. An element is like a single frame in a movie: it represents the UI at a certain point in time.
+Los elementos de React son [inmutables](https://en.wikipedia.org/wiki/Immutable_object). Una vez creas un elemento, no puedes cambiar sus hijos o atributos. Un elemento es como un marco solitario en una película: este representa la Interfaz Gráfica en cierto punto en el tiempo.
```

Comment:
I sugges changing `un marco solitario` with `un fotograma`, given that that's what the original text refers to.

Reply:
I also think that `UI` can be left as is and should be understood anyway.

Reply:
IMHO, translating or not UI is a matter of style guide and should be put into discussion. I think UI has a translation and therefore should be translated

Reply:
@shmesa22 I agree. 
However, wouldn't the translation be `Interfaz de Usuario` or `IU`?